### DISCUSSION TITLE
How to decrease the font size responsive to screen width changes?

### DISCUSSION THREAD

**Discussion by rahulbansal16** [ant-design/ant-design#31508] [Q&A]
Hi, 
I am trying to understand the flow of making the font size responsive based on the screen width.
I have tried the following media querry in the App.less file

`@media only screen and (min-width: 250px) {
  @font-size-base: '25px';
}
`
but this does not increases the font size when the screen size reduces.
How to make font-size responsive in antd and react?
Thanks

**Comment by evgeniyworkbel** [comment]
@rahulbansal16 did you find a way to make responsive text using Ant design?
